principles.md

```diff
@@ -0,0 +1,24 @@
+## Product and Engineering Principles
+
+**We choose simpler designs whenever possible.**
+
+Having less configuration knobs is better for security. Complex products are easier to hack and harder
+to use. Security feature is only good if it make user experience better, otherwise users will find ways to work around it.
+
+That's why we prefer [client certificates](https://goteleport.com/teleport/how-it-works/certificate-based-authentication-ssh-kubernetes/)
+and as way to authenticate and [reduce the number of ports, configuration options and protocols](https://github.com/gravitational/teleport/issues/5777).
+
+**Security through transparency**
+
+The only way to build secure systems is through peer review and transparency.
+
+That's why we have [open design process](https://github.com/gravitational/teleport/pull/5135/files),
```

Comment:
```suggestion
That's why we have [open design process](https://github.com/gravitational/teleport/tree/master/rfd),
```

Reply:
I wanted to show the actual review process with comments, it was interesting for me to see, I think would be interesting for others too